.github/workflows/discord.yml

```diff
@@ -23,9 +23,9 @@ jobs:
 
   notify_push:
     uses: ./.github/workflows/discord_push.yml
-    if: github.ref == 'refs/heads/main' && github.event_name == 'push' && !startsWith(github.event.head_commit.message, 'Merge pull request')
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
```

Comment:
We need this check here because the workflow is used for both `push` and `pull_request` triggers.